test/Microsoft.ML.Tests/Transformers/WordEmbeddingsTests.cs

```diff
@@ -25,7 +23,7 @@ public WordEmbeddingsTests(ITestOutputHelper helper)
         public void TestWordEmbeddings()
         {
             var dataPath = GetDataPath(TestDatasets.Sentiment.trainFilename);
-            var data = new TextLoader(Env,
+            var data = new TextLoader(ML,
```

Comment:
>new TextLoader [](start = 23, length = 14)

you don't seem to like ML.Data.CreateTextLoader :)

Reply:
Well, I'm not sure. I haven't tried to use it *much* yet. So I might like it. But the part below where we're opening a channel so we can call `DataSaverUtils.SaveDataView` struck me as really bad where I felt the need to fix it right then.

---
In reply to: [254556100](https://github.com/dotnet/machinelearning/pull/2453#discussion_r254556100) [](ancestors = 254556100)